demisto_sdk/tests/test_files/integration-Zoom-v1.0.yml

```diff
@@ -0,0 +1,409 @@
+commonfields:
+  id: Zoom-v1.0
+  version: -1
```

Comment:
can we avoid adding new test files?
can you use `TestSuite` here instead maybe?

Reply:
Fixed on c9d835df9f7357f35ef6dac69b6e88e9054fb9ae